rest/taskana-rest-spring/src/test/resources/asciidoc/rest-api.adoc

```diff
@@ -293,6 +293,97 @@ include::{snippets}/TransferTaskDocTest/http-response.adoc[]
 The response-body is essentially the same as for getting a single task. +
 Therefore for the response fields you can refer to the <<task, single task>>.
 
+== TaskComment-Resource
```

Comment:
I did not verify that the snippes actually exist. The build will do that for us :)